adapters/repos/db/shard_read.go

```diff
@@ -188,6 +190,14 @@ func (s *Shard) objectSearch(ctx context.Context, limit int,
 
 func (s *Shard) objectVectorSearch(ctx context.Context, searchVector []float32,
 	limit int, filters *filters.LocalFilter, underscore traverser.UnderscoreProperties) ([]*storobj.Object, error) {
+	defer func() {
+		err := recover()
+		if err != nil {
+			fmt.Printf("\n\n%v\n\n", err)
+			debug.PrintStack()
+		}
+	}()
```

Comment:
this recover part, should be left here?

Reply:
you're absolutely right, that's not meant to be here.